examples/webvr_cubes.html

```diff
@@ -127,7 +127,7 @@
 
```

Comment:
forget it, I think the judder was really just due to naturally low FPS (raycasting against all those boxes I guess).  If I lower the number of boxes in the examples it is nice and smooth, whether `preserveDrawingBuffer` is set or not.  `webvr_stereo_pano` is also working well without it.